etc/delphix-platform/upgrade/unpack-image

```diff
@@ -28,6 +28,11 @@ function usage() {
 	exit 2
 }
 
+function report_progress_inc
```

Comment:
This needs to adhere to the “shfmt” style.. we enforce this with the PR checks that are failing right now. In that test’s output, it’ll show you how to fix the style..